NEWS.md

```diff
@@ -195,6 +199,9 @@
  * \[3.3.0.dev6\] Copying search result filenames causes crash ([#2781](https://github.com/nicotine-plus/nicotine-plus/issues/2781))
  * Recursive folder download matches extra siblings starting with the same name ([#2782](https://github.com/nicotine-plus/nicotine-plus/issues/2782))
 
+You can run the latest unstable build of Nicotine+ to test recent changes and bug fixes, see [TESTING.md](doc/TESTING.md).
+
+If you want to download the current stable version of Nicotine+, see [DOWNLOADS.md](doc/DOWNLOADS.md).
```

Comment:
Not sure if it's a good idea to add this down here. Feels like something someone will eventually forget to remove/update.

Reply:
Good point. Done.